src/Functions/FunctionsBitmap.h

```diff
@@ -460,9 +460,24 @@ struct BitmapSubsetLimitImpl
     }
 };
 
+struct BitmapSubsetOffsetLimitImpl
+{
+public:
+    static constexpr auto name = "bitmapSubsetOffsetLimit";
```

Comment:
Maybe it will be easier to print short name, for example, `subBitmap` (amosbird suggestion)

Reply:
ok